.sonarcloud.properties

```diff
@@ -0,0 +1,10 @@
+sonar.organization=defra
+sonar.projectKey=DEFRA_ffc-demo-payment-service
+sonar.projectName=ffc-demo-payment-service
+sonar.projectVersion=2.0.0
```

Comment:
Shall we remove this, `sonar.projectVersion` seems to only be for Maven projects. https://sonarcloud.io/documentation/analysis/analysis-parameters/

Reply:
I believe the value put into this field will put into the version on the main analysis page, which currently displays as the default if empty `not provided`. For Maven projects it will set it to the version in the Maven pom.xml.

It's currently set to `2.0.0` which doesn't match the package.json version, and it's going to be difficult keeping it in sync as it's not automated, so it may be best to remove it so there is no version, rather than an incorrect one.